tests/storage/lvm_test.py

```diff
@@ -2292,7 +2342,7 @@ def make_pv(pv_name, vg_name):
 
 def make_vg(vg_name, pvs):
     return lvm.VG.fromlvm(
-        "uuid",
+        uuid.uuid4(),
```

Comment:
uuid.uuid4() is not a string. It may work by accident, but always use str(uuid.uuid4()) in the code and the tests.

Switching the code to use uuid.UUID object instead of strings can make sense but this is a huge change so it is unlikely to happen.

Reply:
Good catch. I might revert this actually, I made it because I was looking for more 'complex' scenarios with fake devices, with more fake VGs, etc. And current flow works for 1 VG with 1 PV only. But I ended up removing those tests, this change does not make sense anymore AND probably we will not ever need such complex scenarios anyway.